File: DAY1/성욱/숫자의 개수 - 2577.js
const input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n');

const [A, B, C] = input.map(Number);

const result = (A * B * C).toString();
const counts = new Array(10).fill(0);

for (let digit of result) {
    counts[digit]++;
}

console.log(counts.join('\n'));

File: DAY1/원정/숫자의_개수_2577.py
a = int(input())
b = int(input())
c = int(input())

num = str(a * b * c)

dic = {}

for i in range(0, 10):
    dic[i] = 0

for i in num:
    number = int(i)
    dic[number] += 1
    
for i in dic:
    print(dic[i])